### ISSUE TITLE
Change Screen Resolution on live/installer setup

### ISSUE THREAD

**Issue by G3NSVRV** [arthurchoung/HOTDOG#15]
Hi,
I installed hotdog using the live CD, but far as I look on the files, I could not find any xorg files, .xinintrc, even the hotdot-screenSomething scripts.

I tried to use `xrandr -s 1280x800` but the top bar does not like it:

![image](https://github.com/arthurchoung/HOTDOG/assets/11720535/ec5c7e39-d404-4385-b997-a7dfc3747a8f)

Should I download this repo and run the scripts?
Which is the best approach on my case?

**Comment by arthurchoung** [comment]
A quick hack would be to edit the file '/etc/HOTDOG/Temp/listMonitors.txt'. This file should contain the monitors detected by xrandr. If you edit with 'width:' parameter it should change the width of the menu bar. The menu bar should automatically reload the file.

The wallpaper might still look funny though. You could try selecting 'Enter Aqua Mode' from the menu again, or perhaps run 'hotdog-setupWindowManagerMode.sh'.

A longer explanation is that the script 'hotdog-setupMonitors.pl' is run every time the window manager starts, and every time a monitor is plugged in or unplugged. That script will run 'hotdog-generateSetupMonitorsScriptFromMonitorsFile.pl' and then run the generated script which is a bunch of xrandr commands. There is another script 'hotdog-listMonitors.pl' which will generate the aforementioned 'listMonitors.txt' file. It is somewhat of a mess, I don't remember exactly how it works off the top of my head, but the scripts can be changed.

Ideally the way it would work is that you would choose the resolution from the monitor menu, which is generated by 'hotdog-generateMenuForMonitor:path:.pl'. This hasn't been implemented yet, right now it will only list the possible resolutions.

**Comment by arthurchoung** [comment]
Yes that's the one. The menu is being generated by 'hotdog-generateMenuForMonitor:path:.pl'. It simply runs 'xrandr' and parses the output. The options are blacked out because there is no '=messageForClick=' parameter.

In 'About This Computer > Monitors', that is simply the output from xrandr. The actual script is 'hotdog-aboutMonitors.pl'.
